libs/wlgen/wlgen/rta.py

```diff
@@ -133,7 +133,8 @@ def calibrate(target):
                 pload[cpu] = int(pload_match.group(1))
                 log.debug('>>> cpu%d: %d', cpu, pload[cpu])
 
-        # target.cpufreq.load_governors()
+        for cpu, governor in old_governors.iteritems():
+            target.cpufreq.set_governor('cpu{}'.format(cpu), governor)
```

Comment:
Here you can use the `int` directly, perhaps the documentation should be updated in devlib but it does already that conversion inside.